sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Plugin.java

```diff
@@ -36,6 +36,7 @@
   private String category;
   private String issueTrackerUrl;
   private String sourcesUrl;
+  private boolean supported = false;
```

Comment:
I would be more explicit here. supportedBySonarSource "supported" could mean supported in a given SQ server version.